mods/ts/rules/aircraft.yaml

```diff
@@ -287,3 +287,44 @@ APACHE:
 	RenderSprites:
 	Hovers:
 
+HUNTER:
+	Inherits@1: ^GainsExperience
+	Inherits@2: ^ExistsInWorld
+	Valued:
+		Cost: 1000
+	Tooltip:
+		Name: Hunter-Seeker Droid
+	Health:
+		HP: 500
+	Armor:
+		Type: Light
+	DemoTruck:
+	Explodes:
+		Weapon: SuicideBomb
+		EmptyWeapon: SuicideBomb
+	Aircraft:
```

Comment:
The H-S shouldn't be controllable at all, but rewriting our AutoTarget and DemoTruck logic is way out of scope for this PR.